src/plugins/workspace/public/plugin.ts

```diff
@@ -56,6 +62,9 @@ export class WorkspacesPlugin implements Plugin<{}, {}> {
       }
     }
 
+    /* register workspace column into saved objects table */
+    savedObjectsManagement?.columns.register(getWorkspaceColumn(core));
```

Comment:
Add workspace column for both in/out workspace scenario. Within workspace, saved objects table show both public and workspace specific objects, use this column to differentiate them.